website/docs/components/table/table/partials/guidelines/guidelines.md

```diff
@@ -210,9 +202,8 @@ For more details around using a multi-select Table, recommended patterns, and in
 This example covers basic multi-select table interactions. We're working to include more complex patterns for handling actions when multiple rows are selected and how this impacts filtering and pagination within a data set. For questions or concerns, please reach out to [#team-design-systems](https://hashicorp.enterprise.slack.com/archives/C7KTUHNUS)
 !!!
 
-- When individual rows are selected, the parent checkbox in the Table header changes to display an indeterminate state. 
-- When no or some rows (but not all) are selected in a single Table, clicking the parent checkbox in the Table header will change to display as checked and all rows on that page will be selected.
-- When all rows are selected in a single Table, the parent checkbox in the header appears as checked. Clicking the parent checkbox will deselect all rows on that page.
```

Comment:
This content should be included as it's part of the interaction pattern that isn't defined by the above bullet points. Though it could stand to be updated to something like:

> When all rows are selected, clicking "Select all" in the header will uncheck the checkbox, and all rows on that page will be deselected.